build02/nixpkgs-update-backup.nix

```diff
@@ -0,0 +1,46 @@
+{ config, ... }:
+{
+  # 100GB storagebox is under the build03 hetzner account
+
+  sops.secrets.hetzner-build03-borgbackup-ssh = { };
+
+  programs.ssh.knownHosts = {
+    "hetzner-storage-box" = {
+      hostNames = [ "[u348918.your-storagebox.de]:23" ];
+      publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIICf9svRenC/PLKIL9nk6K/pxQgoiFC41wTNvoIncOxs";
+    };
+  };
+
+  systemd.services.borgbackup-job-nixpkgs-update.serviceConfig.ReadWritePaths = [
+    "/var/log/telegraf"
+  ];
+
+  services.borgbackup.jobs.nixpkgs-update = {
+    paths = [
+      "/var/log/nixpkgs-update"
+    ];
+    repo = "u348918@u348918.your-storagebox.de:/./nixpkgs-update";
+    encryption.mode = "none";
```

Comment:
I don't think we need to bother with encryption?

Reply:
Probably yes since the logs are public.